### DISCUSSION TITLE
Entire Project Pages DELETED

### DISCUSSION THREAD

**Discussion by alexander-hakim** [community/community#67639] [Repositories]
### Select Topic Area

Bug

### Body

I have 5 Github Projects under an org.

2 of them are private, 3 are public. Every single one of them just had their content deleted except for 1 card (i believe the oldest card created on that project). 

There is no version control for projects, so I cannot go back and see my previous content. Other projects that are not in that repo also were deleted (private ones on my account). 

Is anyone else having this issue? I do not know how to download my old content for projects, or if this is an issue with Github as a whole today.

**Comment by AdamYoblick** [comment]
I'm seeing this too, and I see other discussions about this topic. Seems like a known issue.

**Reply by trungnhm1998** [reply]
Mine too, 8 sprints over 500 cards just gone this morning still there 6 hours ago.
